public/index.html

```diff
@@ -3787,6 +3787,15 @@ <h4 class="title_restorable">
                         <div name="themeElements" data-newbie-hidden class="flex-container flexFlowColumn flexNoGap">
                             <!-- <h4><span data-i18n="UI Colors">Theme Settings</span></h4> -->
                             <div name="AvatarAndChatDisplay" class="flex-container flexFlowColumn">
+                                <div class="flex-container">
+                                    <span data-i18n="Chat Mode:">Chat Mode:</span><br>
+                                    <select id="chat_mode" class="widthNatural flex1 margin0">
+                                        <option value="0" data-i18n="Normal">Normal</span>
+                                        <option value="1" data-i18n="Visual Novel (VN)">Visual Novel (VN)</option>
+                                        <option value="2" data-i18n="VN w/ Vertical Letterbox">VN w/ Vertical Letterbox</option>
```

Comment:
In Spanish you had no abbreviations either. I think that "w/" is a bit confusing.
I'd choose either of those:
- VN (Vertical Letterbox)
- VN with Vertical Letterbox